addons/website_slides_survey/models/slide_slide.py

```diff
@@ -65,3 +65,28 @@ def _action_set_viewed(self, target_partner, quiz_attempts_inc=False):
                         'slide_partner_id': new_slide_partner.id
                     }
                 )
+
+    def _get_slide_certification_data(self):
+        certification_data = {}
+        if not all(slide.channel_id.is_member for slide in self):
+            for s in self:
```

Comment:
Please avoid using one letter variable name (only in lambdas and even there I'm not a huge fan).

Reply:
Done

Reply:
Done